### PR TITLE
Kdh step1

### PR DESCRIPTION
**Pull request by kor-haru**
my step1 solution review plz

### CODE REVIEW

**Review by jukqaz — CHANGES_REQUESTED** [review]
@dopingcat 
I got it.

**Comment by kor-haru** [comment]
@jukqaz remove js source complete

**Comment by jukqaz** [comment]
@dopingcat 
Good 👍 , checked this.

**Review by jukqaz — APPROVED** [review]

**Review by cjy9306 — APPROVED** [review]
congratulations

File: 1/kdh/2_1000.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import sys
+
+for line in sys.stdin:
+	parsedInput = line.split()
```
Comment (jukqaz) [inline 1/kdh/2_1000.py:4]: @dopingcat 
Python is very nice.

Comment (kor-haru) [reply 1/kdh/2_1000.py:4]: Python is simple and good :)

**Comment by jukqaz** [comment]
@korJiusngPark @AlvinYou 
Review Please.

**Review by korJiusngPark — APPROVED** [review]
👍

**Review by AlvinYou — APPROVED** [review]
👍

**Pull request merged**